app/static/js/grid_scripts.js

```diff
@@ -0,0 +1,48 @@
+const tolerance = 1;
+let last = 0;
+let lastIndex = 0;
+
+function setToggle(elementClass, targetElement, classToAdd) {
+    let allDays = document.getElementsByClassName(elementClass);
+    for (let i = lastIndex; i < allDays.length; ++i) {
+        allDays[i].addEventListener("click", function () {
+            let target = document.getElementById(targetElement);
```

Comment:
const

Reply:
True, what about the i - from what I understand it needs to be let cus it changes over the loop.